.github/workflows/ci.yml

```diff
@@ -38,6 +38,27 @@ jobs:
             database_image: "mariadb:11.2"
             coverage: true
             experimental: false
+          - mediawiki_version: '1.41'
+            smw_version: dev-master
+            php_version: 8.1
+            database_type: mysql
+            database_image: "mariadb:11.2"
+            coverage: false
+            experimental: false
+          - mediawiki_version: '1.42'
+            smw_version: dev-master
+            php_version: 8.1
+            database_type: mysql
+            database_image: "mariadb:11.2"
+            coverage: false
+            experimental: false
+          - mediawiki_version: '1.43'
+            smw_version: dev-master
+            php_version: 8.1

```

Comment:
if we like, we can add "pf_version: xxx" here and to the env belown

Reply:
👍

Reply:
So you want it set for just MW 1.41/42/43? or for all?

Reply:
If we add it to the matrix I would set it for all matrix cases. If it's the same version for all at the moment, it's okay.

Reply:
Thanks, done.

Reply:
awesome!